docs/intro.md

```diff
@@ -1,27 +1,21 @@
 ---
 id: intro
-title: Table of contents
+title: Introduction
 sidebar_position: 1
 slug: /
 description: Contents of (course notes title here)
 ---
 
-# Table of contents
+# Introduction
 
-## Introduction <a id="part-a-introduction"></a>
+This is a demonstration of a docusaurus template that we use to create OER's.
 
-- [Test Page](A-Introduction/testfile.md)
-- [Topic 2](A-Introduction/topic-file-2.md)
-- [Topic 3](A-Introduction/topic-file-3.md)
+## PDF version of these notes:
 
-## SubSection2
+A pdf version of these notes can be found here:
 
-- [Topic 1](B-SubSection2/topic-file-1.md)
-- [Topic 2](B-SubSection2/topic-file-2.md)
-- [Topic 3](B-SubSection2/topic-file-3.md)
+https://seneca-ictoer.github.io/OERTemplate/pdf/OERTemplate.pdf
```

Comment:
Can we not link to the PDF like a regular asset file (relative link) so this won't break in other projects that use a different URL?

Reply:
I tried looking at how to do this when I was originally working with this, but I think if I remember correctly the issue was that once it was made an asset file, it was unclear how to get the pdf link to that asset automatically as there was some randomness to the name.  I agree that this is probably not the best way to do it.  It just ensures that it is in a known location each time.

Reply:
oh.. I am completely misreading this.. Let me think about this and try it out.

Reply:
The relative link is showing an odd bug. I think I know why.. but uncertain of solution.
This video shows what is happening:

https://youtu.be/iSr0-aX_LKo

Basically, when you use the link from the page it shows as a "page not found" error.  Pretty certain this is generated from docusaurus.  But when you reload the page url you can get to the pdf.  

My guess is the error occurs because on the relative link docusaurus might be trying to add it as if the pdf folder was a subfolder of docs.  there is no pdf/ folder within docs/ and therefore docusaurus treats it as a broken page and the error page is generated.

The pdf is generation occurs after the docusaurus site gets created because we want to generate the pdf from the most recent version of the site, so we don't want to do the generation before hand.

Not sure of how to fix this yet.

Reply:
Did a bit more investigation.. for assets that are linked from an internal source such as an image in the src/img folder, the src of the image tag was rendered as: 
```
data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAMgAAADICAYAAACtWK6eAAAT3UlEQVR42u1dCVQVV5pWXNt2N0czykl33KImZ7IgKgqIghq3KCDK+qowCek2c2K0Mx3idBxakzYxJnZiq3Gf6Bg7UdN2R51MxnTSia3gew9Rwccm7oqiiIK4sPxTt1hEHo9XvPVW1fed852Dr+67UNb/1f3/+9/731atAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAO8i2CxGjDUJXzMGmcSZnmoHAF7B6GMJvYPNwq5gk1AmMS/YJMbaahtkNsRLbeghmoU4d7cDAO+NCEbhQCMjrZbe5q81bhdyVOwuXbtqZdDSZ+yau9oBgNc...
```
If I put a dummy pdf into the src folder like an image, the updated pdf based on most recent version of site probably won't work as it is likely to be rendered in a similar manner.. (not certain, have not tried yet.. just writing down possible solutions)